web/packages/teleport/src/Integrations/helpers.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { LabelKind } from 'design/Label/Label';
+
+import {
+  getStatusCodeTitle,
+  Integration,
+  IntegrationStatusCode,
+} from 'teleport/services/integrations';
+import { IntegrationLike, Status } from 'teleport/Integrations/types';
+
+export function getStatus(item: IntegrationLike): Status | null {
```

Comment:
This function doesn't return a null value.

Reply:
You're right, I moved this method from `web/packages/teleport/src/Integrations/IntegrationList.tsx` to this file without change, but I'll go ahead and update the type here.